packages/terra-clinical-item-display/src/terra-dev-site/doc/clinical-item-display/clinicalItemDisplay.1.doc.mdx

```diff
@@ -52,6 +54,17 @@ import ItemDisplay from 'terra-clinical-item-display';
 <DefaultComment title="Comment Item Display" />
 <TextStyleMeaning title="Item Display: Text Style Meaning for Assistive Technology" />
 
+<Notice variant="important" ariaLevel="3">
+
+#### Accessibility Guidance: Truncated Text
+
+When using `isTruncated`, consumers are responsible for providing a pattern to disclose the full Item Display text
```

Comment:
What does `pattern` mean here?

Reply:
A method/way to disclose the text. I just used the word 'pattern' because it's used in the Jira description. "Method" is probably easier to understand - will update.

Reply:
Updated: [485e05e](https://github.com/cerner/terra-clinical/pull/848/commits/485e05ef4dc443613b68e587f5f0973aa11bd1c2)